docs/pocs-alternatives.rst

```diff
@@ -0,0 +1,96 @@
+=================
+POCS Alternatives
+=================
+
+A primary software adage is to avoid "recreating the wheel" and while automated OCS
+systems are not unique, an initial review found that none of the available systems were
+suitable to the PANOPTES goals outlined in the `PANOPTES Overview <panoptes-overview.html>`_. First, all software that
+required license fees or was not otherwise free (of cost) and open (to modification) was automatically rejected.
```

Comment:
c/automatically rejected/not applicable/

A softer way of putting it, and doesn't sound like a knee-jerk response.

Reply:
This is all lifted directly from (a draft) of my thesis, and I thin I was trying to be assertive. ;)